docs/guide/core-widgets.md

```diff
@@ -13,7 +13,7 @@ Apostrophe comes with some content widgets you can use in areas right away. See
 
 The rich text widget provides a space for entering and editing formatted text. Editors can update its content directly in-context.
 
-There are many text formatting features that you can configure for rich text widgets. These editor options are configured in two widget options: [`toolbar`](#configuring-the-toolbar) and [`styles`](#configuring-text-styles). Add these to the widget configuration object when adding an area field.
+There are many text formatting features that you can configure for rich text widgets. These editor options are configured in three widget options: [`toolbar`](#configuring-the-toolbar), [`styles`](#configuring-text-styles), and [`linkWithType`](#allowing-links-to-specific-piece-types). Add these to the widget configuration object when adding an area field. 
```

Comment:
linkWithType is an option to the `@apostrophecms/rich-text-widget` **module**. It is not an option to an **individual area**. Therefore it is configured in a different place and shouldn't be shown in the same example code or in the same sentence.

Reply:
The setting for this option governs internal links in **all** areas throughout the site. (Otherwise it would be very difficult for Apostrophe to quickly populate them when loading pages.)

Reply:
Refactored out to its own section.